addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
```

Comment:
Cool - Don't know where it get's turned on, but our earlier tests did not have an AU chart, and so AU currency never got turned on!  Just tested without the active, and it worked. ;)